src/views/AnnualReport.vue

```diff
@@ -859,9 +866,17 @@ export default {
           }
           filing = res.data.filing
           this.haveChanges = false
-        }).catch(error => {
+        }).catch(async error => {
           if (error && error.response && error.response.status === PAYMENT_REQUIRED) {
-            this.paymentErrorDialog = true
+            const errCode = this.getErrorCode(error)
+            if (errCode) {
+              const errObj = await this.getErrorObj(errCode.payment_error_type)
+              if (errObj) {
+                this.bcolObj = errObj
```

Comment:
This "if" is not needed. It looks like it assigns errorObj if it's not null, which is the same thing that would happen without the if... unless you want to set bcolObj=null in an else block?

Reply:
done

Reply:
Fixed.